src/js/components/HOCS/withBarHOC.js

```diff
@@ -13,9 +115,17 @@ function calculaAcumulado(valores = []) {
     return Util.formataParaReal(total);
 }
 
-function getFormattedLabel(tooltipItem, data) {
-    const { label, data: dataValues } = data.datasets[tooltipItem.datasetIndex];
+function encontraLegenda(label) {
+    const legendaEncontrada = legendas.find(legenda => legenda.label === label);
 
+    return legendaEncontrada && legendaEncontrada.legenda
```

Comment:
here we can simplify to `legendaEncontrada ? legendaEncontrada.legenda : ''` what do u think?

Reply:
ok